### PR TITLE
Improve metadata descriptions

### PR DESCRIPTION
**Pull request by AlexJerabek**
According to the CATS report, the descriptions for these pages were not of sufficient length. They have been adjusted accordingly.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[836cbae](https://github.com/OfficeDev/office-scripts-docs/commits/836cbaec2795b33328d6e23acdef108b85a1757b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/index.md) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/testing/undo.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/testing/undo.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C8%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001081841348141-73%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FXHqkSr57dnzXOOqvYPWIoCmeYGzWgRQulOpE6rj%2B0g%3D&st=2020-01-08T18%3A37%3A47Z&se=2020-02-08T18%3A42%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Record, edit, and create Office Scripts in Excel on the web'
-description: 'A tutorial teaching the basics of writing and editing Office Scripts.'
+description: 'A tutorial teaching the basics of writing and editing Office Scripts, including reading data from and and writing data to a workbook.'
```
Comment (lindalu-MSFT) [inline docs/tutorials/excel-tutorial.md]:
```suggestion
description: 'A tutorial about the basics of Office Scripts, including reading data from and writing data to a workbook.'
```

**Comment by lindalu-MSFT** [comment]
@AlexJerabek Hi Alex, thanks for requesting my review! I made some suggestions based on improving SEO and eliminating redundancy! LMK if you have questions.

**Comment by AlexJerabek** [comment]
Thanks @lindalu-MSFT! All of your feedback has been addressed.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ad61ae6](https://github.com/OfficeDev/office-scripts-docs/commits/ad61ae6ddbfa833d50b0bd3f3267fd27b343606d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/index.md) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/testing/undo.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/testing/undo.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C8%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001081916084339-73%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=myVt67UH2k7acHxCSavJ7dRjcdTc1GhU75JYY8WP4K4%3D&st=2020-01-08T19%3A12%3A18Z&se=2020-02-08T19%3A17%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[90b04e2](https://github.com/OfficeDev/office-scripts-docs/commits/90b04e2654549225162555df01ce8f0f7f03cec9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/index.md) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/testing/undo.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/testing/undo.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C8%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001081916441770-73%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZTD3r%2F381WkOmW2Ih8nnRn8Ukp5gCFZlnJFaJHM8jls%3D&st=2020-01-08T19%3A13%3A36Z&se=2020-02-08T19%3A18%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d6e1c1b](https://github.com/OfficeDev/office-scripts-docs/commits/d6e1c1bac082c2e2582cc3cb6743961eeaa8604a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/index.md) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/testing/undo.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/testing/undo.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-CATS/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001091821188356-73%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=quXMUf7L%2Bt96b1EWcB0KK6DksNAXcbyO2RssnszFfzw%3D&st=2020-01-09T18%3A17%3A31Z&se=2020-02-09T18%3A22%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Requested changes were made. Checking in.

**Pull request merged**